blockchain/src/index.js

```diff
@@ -350,9 +355,7 @@ app.post("/chain", async (req, res) => {
           config.DirectoryHash,
           extractPath,
         );
-        const chainConfig = yaml.safeLoad(
-          fs.readFileSync(chainConfigPath, "utf8"),
-        );
+        const chainConfig = yaml.load(fs.readFileSync(chainConfigPath, "utf8"));
```

Comment:
pls do not replace `load` by` safeLoad` more information [here](http://adilapapaya.com/docs/js-yaml/#safeloadstringoptions)